apps/openchallenges/organization-service/src/test/java/org/sagebionetworks/openchallenges/organization/service/OrganizationServiceApplicationTests.java

```diff
@@ -6,6 +6,7 @@
 @SpringBootTest
 class OrganizationServiceApplicationTests {
 
+  // prepare the context for the unit tests
```

Comment:
Don't take my words for it. If unsure about what this line does, remove this comment.

Reply:
Ok I'll just remove the comment.